articles/azure-monitor/app/opencensus-python.md

```diff
@@ -29,7 +28,7 @@ Sign in to the [Azure portal](https://portal.azure.com/).
 
 ## Create Application Insights resource
 
-First you have to create an Application Insights resource which will generate an instrumentation key(ikey). The ikey is then used to configure the local forwarder to send distributed traces from your OpenCensus instrumented application, to Application Insights.   
+First you have to create an Application Insights resource which will generate an instrumentation key(ikey). The ikey is then used to configure the OpenCensus SDK to send telemetry data to Application Insights.
```

Comment:
Nit: You should be consistent with the "First" terminology. Your proceeding steps should start with "Second, Thirdly, etc..." or don't use it at all.